docs/features/proposals/collection-of-from.md

```diff
@@ -0,0 +1,96 @@
+# [`.of` and `.from` methods on collection constructors](https://github.com/tc39/proposal-setmap-offrom)
+Modules [`esnext.set.of`](/packages/core-js/modules/esnext.set.of.js), [`esnext.set.from`](/packages/core-js/modules/esnext.set.from.js), [`esnext.map.of`](/packages/core-js/modules/esnext.map.of.js), [`esnext.map.from`](/packages/core-js/modules/esnext.map.from.js), [`esnext.weak-set.of`](/packages/core-js/modules/esnext.weak-set.of.js), [`esnext.weak-set.from`](/packages/core-js/modules/esnext.weak-set.from.js), [`esnext.weak-map.of`](/packages/core-js/modules/esnext.weak-map.of.js), [`esnext.weak-map.from`](/packages/core-js/modules/esnext.weak-map.from.js)
+```ts
+class Set {
+  static of(...args: Array<mixed>): Set;
+  static from(iterable: Iterable<mixed>, mapFn?: (value: any, index: number) => any, thisArg?: any): Set;
+  addAll(...args: Array<mixed>): this;
+  deleteAll(...args: Array<mixed>): boolean;
+  every(callbackfn: (value: any, key: any, target: any) => boolean, thisArg?: any): boolean;
+  filter(callbackfn: (value: any, key: any, target: any) => boolean, thisArg?: any): Set;
+  find(callbackfn: (value: any, key: any, target: any) => boolean), thisArg?: any): any;
+  join(separator: string = ','): string;
+  map(callbackfn: (value: any, key: any, target: any) => any, thisArg?: any): Set;
+  reduce(callbackfn: (memo: any, value: any, key: any, target: any) => any, initialValue?: any): any;
+  some(callbackfn: (value: any, key: any, target: any) => boolean, thisArg?: any): boolean;
```

Comment:
They should be in `collection-methods`.

Reply:
19f2464e9ef9149eef13fc9b02fa3e3951b34661

Reply:
@Santiagosaiz I don't understand. Why is that commit related to this?